instrumentation/rw_test.go

```diff
@@ -0,0 +1,14 @@
+package instrumentation
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/fd/httpmiddlewarevet"
```

Comment:
I really like the tool. Does it check what happens if the basic ResponseWriter is passed to a middleware? As I understand it check if the created middleware conforms to http.* interfaces. What about the other way: what happen if you pass something without the interfaces implemented.